ohdear/resource_site.go

```diff
@@ -30,15 +29,39 @@ func resourceOhdearSite() *schema.Resource {
 				ForceNew:    true,
 				Description: "ID of the team for this site",
 			},
-			"checks": &schema.Schema{
-				Type:        schema.TypeSet,
+			"uptime": &schema.Schema{
+				Type:        schema.TypeBool,
 				Optional:    true,
-				Description: "Checks to include for side, default is all checks. Note: you cannot enable certificate checks on http URLs.",
-				Elem: &schema.Schema{
-					Type:         schema.TypeString,
-					ValidateFunc: validation.StringInSlice(ohdear.CheckTypes, false),
-				},
+				Description: "Enable/Disable uptime check",
+				Default:     true,
 			},
+			"broken_links": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Enable/Disable broken_links check",
+				Default:     true,
+			},
+			"certificate_health": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Enable/Disable certificate_health check",
+				Default:     true,
+			},
+			"mixed_content": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Enable/Disable mixed_content check",
+				Default:     true,
+			},
+			"certificate_transparency": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Enable/Disable certificate_transparency check. Cannot be used with http URLs",
+				Default:     true,
```

Comment:
I think if you wrote a test with an `http` URL and left the checks default, you would get a perpetual diff since the API will return false but the config will be defaulted to true. You'd always have to manually set it to `true`.

Reply:
True dat - I'll add a test case for this and make sure its :100: